### ISSUE TITLE
Create empty main window with tabs (1)

### ISSUE THREAD

**Issue by JoeZiminski** [neuroinformatics-unit/datashuttle#185]
The first step in the TUI is to create a main window app page that will hold a tab widget, with pages 'Create' and 'Transfer'. The aim is for switching tabs to switch the page in the area underneath (image below). It seems Textual has already combined Tabs and ContentSwitcher into a [TabbedContent](https://textual.textualize.io/widgets/tabbed_content/) widget which seems to be exactly what we want.

We can include simple text on each content window that just says 'create' and 'transfer' so we know the tab switching is working well.  Once this is done we can start building the 'Create' page.

Even at this early stage it is probably worth separating the css out into separate files (even if these are currently blank). We can follow the example at [ContentSwitcher](https://textual.textualize.io/widgets/content_switcher/#__tabbed_1_2) (where `CSS_PATH = "content_switcher.tcss"` and they show the example .css file). We can keep this blank or add one field to check it is working. If your happy with it, we can go with a structure such as:

```
.
└── datashuttle/
    ├── pyproject.toml
    ├── ....
    └── datashuttle/
        ├── configs
        ├── ...
        └── tui/
            ├── css/
            │   └── content_switcher.css
            └── view/
                └── app.py  # contains the app / content switcher
```
We can figure out exactly what to call these folders / files later when their function is clearer.

Image ( for now, lets ignore the 'Setup' tab, and the top bar with datashuttle and the minimize / restore / close buttons).
![mainwindow](https://github.com/neuroinformatics-unit/datashuttle/assets/55797454/e2870969-98cc-4595-9241-bd7dc6b2f7f5) 

To summarize, we can:

1) Create an initial app with TabbedContent widget with tabs 'Create' and 'Transfer'
2) add the text 'create' and 'transfer' to the content area, so that switching tabs switches the text.
3) create a `.css` file for the TabbedContent widget and add one field (e.g. the first feild from the example) to check it works.

Please let me know if there are any questions or anything is not clear!

**Comment by b-peri** [comment]
Hey @JoeZiminski! I've whipped together a minimal TUI with functioning tabs and a basic CSS file now. 

If you're interested in playing around with the app & the CSS, you might like to know that Textual has some fun features such as live CSS editing. Just run the app from terminal using ```textual run --dev app.py``` (assuming you have the ```textual-dev``` package installed) and any changes you make in ```tab_content.tcss``` will be visible in the terminal moments later!
